libs/postgres_ffi/wal_craft/Cargo.toml

```diff
@@ -15,3 +15,7 @@ postgres_ffi.workspace = true
 tempfile.workspace = true
 
 workspace_hack.workspace = true
+
+[dev-dependencies]
+regex.workspace = true
+utils.workspace = true
```

Comment:
Newline needed at the end; you can probably fix your editor to always output these, or watch git cry about whitespace errors :)